hcl2template/types.build.provisioners.go

```diff
@@ -167,3 +168,36 @@ func (cfg *PackerConfig) startProvisioner(source SourceBlock, pb *ProvisionerBlo
 	}
 	return provisioner, diags
 }
+
+type provisionerPrepare struct {
+	cfg               *PackerConfig
+	provisionersBlock []*ProvisionerBlock
```

Comment:
```suggestion
	provisionerBlocks []*ProvisionerBlock
```

Reply:
oh nice! I just copied and pasted it and forgot to update the names 🤦